src/Webapi/Dom/Webapi__Dom__HtmlOptionsCollection.res

```diff
@@ -0,0 +1,30 @@
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#htmloptionscollection
+ */
+type t = Dom.htmlOptionsCollection
+
+// realistically this only deals with HtmlOption and HtmlOptGroup elements
```

Comment:
what do you all think of reimplementing the `HtmlCollection` functions in here to return `Dom.htmlOptionElement` instead of doing the include?

Reply:
There aren't many bindings in `HtmlCollection` so yes I think that's fine. The only point of the include is to reduce 
copy/pasting bindings but it does come at a cost (the minimum size of a Functor is a function call and an empty object neither of which can be tree shaken).